src/tables/tablesRestApiClient.ts

```diff
@@ -87,4 +88,30 @@ export default class TablesRestApiClient extends TablesApiClient {
       throw new MindsDbError(sqlQueryResult.error_message);
     }
   }
+
+  /**
+   * Uploads a file to a specified table in an integration.
+   * @param {string} filePath - Path to the file to be uploaded.
+   * @param {string} tableName - Name of the table to upload the file to.
+   * @param {string} integration - Name of the integration the table is a part of.
+   * @throws {MindsDbError} - Something went wrong uploading the file.
+   */
+  override async uploadFile(filePath: string, tableName: string, integration: string): Promise<void> {
+    const fileName = path.basename(filePath);
+
+    const sqlQuery = `
```

Comment:
@setohe0909 Does this query actually upload the file from local? Shouldn't we use multipart upload?